cmd/kyma/provision/minikube/cmd.go

```diff
@@ -230,6 +231,7 @@ func (c *command) startMinikube() error {
 		"--vm-driver", c.opts.VMDriver,
 		"--disk-size", c.opts.DiskSize,
 		"-b", bootstrapper,
+		"-profile", c.opts.Profile,
```

Comment:
Usually, when a flag is not a shorthand, it uses double hyphen `--` for its full name.

Reply:
no probs